import mysql.connector
from mysql.connector import error
import pandas as pd

#Testing 2